components/style/properties/properties.mako.rs

```diff
@@ -1504,6 +1297,180 @@ impl PropertyDeclaration {
             PropertyDeclaration::Custom(..) => false,
         }
     }
+
+    /// The `in_keyframe_block` parameter controls this:
```

Comment:
I don't see an `in_keyframe_block` parameter anymore, though I think it's preexisting, mind to update it?